### PR TITLE
Remove duplicate ecf scripts that are just copies of f000 in all cases.

### PR DESCRIPTION
**Pull request by aerorahul**
This PR:
- removes all forecast hour `.ecf` files that are copies of `f000.ecf`.
- introduces a script `setup_ecf_links.sh` that creates soft link of the appropriate forecast hour from the `master.ecf` file.
- `grib_wafs` was using inconsistent forecast hour; padding single digit forecast hour with `0`, but not for 2 digit forecast hours.  This has been rectified.  Corresponding change to the suite definitions have been made.
- removes a remnant from PR #555 that copied `gdas/enkf` to `enkfgdas`
- A total of approximately 365 `.ecf` files have been deleted.  365 less files to maintain.

**Note for the reviewers:**
- Please clone `feature/ops-wcoss2` in a location
- Please clone `feature/nco-remove-duplicate-ecfs` in a **different** location and then do the following:
```
cd ecf
./setup_ecf_links.sh
```
- Please compare the directory structure for filenames, missing files, etc between the 2 locations (specifically `ecf` directory) to make sure that the developer did not accidentally remove a file that is required.

Adding @WeiWei-NCO for his review and feedback

### CODE REVIEW

File: ecf/setup_ecf_links.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+fhrs=($(seq 0 3 84) $(seq 90 6 240))
+link_master_to_fhr "jgfs_atmos_awips_g2" "$fhrs"
+
+# GFS atmos wafs files
+cd $ECF_DIR/scripts/gfs/atmos/post_processing/grib_wafs
+echo "Linking gfs/atmos/post_processing/grib_wafs ..."
+fhrs=($(seq 0 6 120))
+link_master_to_fhr "jgfs_atmos_wafs" "$fhrs"
+
+
```
Comment (WalterKolczynski-NOAA) [inline ecf/setup_ecf_links.sh]: Remove all the extra blank lines at the end.

File: ecf/setup_ecf_links.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+
+
+
+
+
```
Comment (KateFriedman-NOAA) [inline ecf/setup_ecf_links.sh]: Bunch of extra spaces here that can probably go.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Enthusiastically approve, assuming there is no f000 weirdness that has been overlooked.

**Comment by WeiWei-NCO** [comment]
May need to test on my side, but it looks good to me. Thank you.

**Review by KateFriedman-NOAA — APPROVED** [review]
Changes look good and are as described. Ran new link script per developer instructions and compared with auth branch to ensure nothing was missed. Here are the "Only in" instances from a recursive diff between the two:
```
Kate.Friedman@dlogin03:/lfs/h2/emc/global/noscrub/Kate.Friedman/git> diff -r feature-nco-remove-duplicate-ecfs/ecf/scripts/ feature-ops-wcoss2-auth/ecf/scripts/ | grep "Only in"
Only in feature-ops-wcoss2-auth/ecf/scripts/enkfgdas/post: change-task-enkf-post.sh
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/enkfgdas/post: .gitignore
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/enkfgdas/post: jenkfgdas_post_master.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gdas/atmos/post: .gitignore
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gdas/atmos/post: jgdas_atmos_post_master.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gdas: enkf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post: .gitignore
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post: jgfs_atmos_post_master.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/awips_20km_1p0: .gitignore
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/awips_20km_1p0: jgfs_atmos_awips_master.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/awips_g2: .gitignore
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/awips_g2: jgfs_atmos_awips_g2_master.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: .gitignore
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f000.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f006.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f00.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f012.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f018.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f024.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f030.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f036.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f042.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f048.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f054.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f060.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f066.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f06.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f072.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f078.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f084.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f090.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f096.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f12.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f18.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f24.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f30.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f36.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f42.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f48.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f54.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f60.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f66.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f72.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f78.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f84.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f90.ecf
Only in feature-ops-wcoss2-auth/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_f96.ecf
Only in feature-nco-remove-duplicate-ecfs/ecf/scripts/gfs/atmos/post_processing/grib_wafs: jgfs_atmos_wafs_master.ecf
```

File: ecf/setup_ecf_links.sh (hunk 1 above)
Comment (aerorahul) [reply ecf/setup_ecf_links.sh]: done.

**Review by KateFriedman-NOAA — APPROVED** [review]

**Pull request merged**